### PR TITLE
Magento_Paypal: avoid using deprecated escape* methods from AbstractB…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/Paypal/view/adminhtml/templates/billing/agreement/view/tab/info.phtml [hunk 1]
```diff
@@ -3,38 +3,41 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-/** @var \Magento\Paypal\Block\Adminhtml\Billing\Agreement\View\Tab\Info $block */
+/**
```
Comment (ihor-sviziev) [inline app/code/Magento/Paypal/view/adminhtml/templates/billing/agreement/view/tab/info.phtml:7]:
```suggestion

/**
```

File: app/code/Magento/Paypal/view/adminhtml/templates/system/config/api_wizard.phtml [hunk 2]
```diff
@@ -7,6 +7,7 @@
 /**
  * @see \Magento\Paypal\Block\Adminhtml\System\Config\ApiWizard
```
Comment (ihor-sviziev) [inline app/code/Magento/Paypal/view/adminhtml/templates/system/config/api_wizard.phtml]:
```suggestion
```

File: app/code/Magento/Paypal/view/adminhtml/templates/system/config/bml_api_wizard.phtml [hunk 3]
```diff
@@ -7,21 +7,22 @@
 /**
  * @see \Magento\Paypal\Block\Adminhtml\System\Config\BmlApiWizard
```
Comment (ihor-sviziev) [inline app/code/Magento/Paypal/view/adminhtml/templates/system/config/bml_api_wizard.phtml]:
```suggestion
```

File: app/code/Magento/Paypal/view/adminhtml/templates/system/config/payflowlink/info.phtml [hunk 4]
```diff
@@ -7,13 +7,14 @@
 /**
  * @see \Magento\Paypal\Block\Adminhtml\System\Config\Payflowlink\Info
```
Comment (ihor-sviziev) [inline app/code/Magento/Paypal/view/adminhtml/templates/system/config/payflowlink/info.phtml]:
```suggestion
```

File: app/code/Magento/Paypal/view/adminhtml/templates/system/config/payflowlink/advanced.phtml [hunk 5]
```diff
@@ -4,15 +4,16 @@
  * See COPYING.txt for license details.
  */
 
-/*
+/**
  * @var $block \Magento\Paypal\Block\Adminhtml\System\Config\Payflowlink\Advanced
```
Comment (ihor-sviziev) [inline app/code/Magento/Paypal/view/adminhtml/templates/system/config/payflowlink/advanced.phtml]:
```suggestion
 * @var \Magento\Paypal\Block\Adminhtml\System\Config\Payflowlink\Advanced $block
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Paypal/view/adminhtml/templates/system/config/api_wizard.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Paypal/view/adminhtml/templates/system/config/api_wizard.phtml]: fixed

File: app/code/Magento/Paypal/view/adminhtml/templates/system/config/bml_api_wizard.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Paypal/view/adminhtml/templates/system/config/bml_api_wizard.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests CE, Functional Tests EE

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Please resolve conflicts

File: app/code/Magento/Paypal/view/adminhtml/templates/system/config/payflowlink/advanced.phtml (hunk 5 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Paypal/view/adminhtml/templates/system/config/payflowlink/advanced.phtml]: fixed

**Comment by sergeynezbritskiy** [comment]
> Please resolve conflicts

Hello @ihor-sviziev, fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Database Compare, Functional Tests B2B

**Comment by ihor-sviziev** [comment]
@magento run Database Compare

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8914](https://jira.corp.magento.com/browse/ENGCOM-8914) has been created to process this Pull Request

**Pull request closed without merging**